src/Curl.Parser.Net/Models/Parsing/KeyTypeParameterEvaluator.cs

```diff
@@ -14,14 +15,14 @@ public KeyTypeParameterEvaluator()
     protected override void EvaluateInner(ref Span<char> commandLine, ConvertResult<CurlOptions> convertResult)
     {
         var value = commandLine.ReadValue().ToString();
-        if (Enum.TryParse(value, true, out KeyType keyType))
+        if (Enum.TryParse(value, true, out CertificateType keyType))
```

Comment:
We should keep using `KeyType` here, it's not the same as `CertificateType`

Reply:
I see. Updated.